From 46aa3847fa4983f01362cf60479f4f0edad99eff Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Tue, 17 Jun 2014 01:00:31 +0400
Subject: [PATCH 2/7] Fix pc.in template

---
 pc.in | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/pc.in b/pc.in
index 98bbb84..77a4548 100644
--- a/pc.in
+++ b/pc.in
@@ -1,7 +1,8 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/include
 
-Name: @PROJECT_NAME@
-Description: @CPACK_PACKAGE_DESCRIPTION_SUMMARY@
-Version: @APPLICATION_VERSION@
-Cflags: -I@CMAKE_INSTALL_PREFIX@/include/@PROJECT_NAME@
+Name: @PKG_NAME@
+Description: @PKG_DESC@
+Version: @PKG_VERSION@
+Cflags: -I${includedir}
-- 
1.9.1

