From 1447cbdfd437871dcf6c9abdf1814dd5fb94d1e1 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Tue, 17 Jun 2014 02:43:53 +0400
Subject: [PATCH 5/7] Add install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3651b58..3e4c45b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,4 +94,7 @@ configure_file("COPYING" "COPYING.txt" COPYONLY)
 install(FILES ${PROJECT_BINARY_DIR}/COPYING.txt
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/ COMPONENT license)
 
+install(FILES ${CMAKE_SOURCE_DIR}/package.xml
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/ COMPONENT catkin)
+
 # vim:sw=4:ts=4:expandtab
-- 
1.9.1

