From e5ea253417b8ebf584bb54a3ba66209d7dbe1640 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Sat, 21 Jun 2014 15:19:57 +0400
Subject: [PATCH 6/7] Add libdir hack from urdfdom.

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3e4c45b..2f36980 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,6 +14,9 @@ set(PROJECT_CONTACT_VENDOR mavlink)
 
 include(GNUInstallDirs)
 
+# hack from urdfdom: by default this would be 'lib/x86_64-linux-gnu'
+set(CMAKE_INSTALL_LIBDIR lib)
+
 # find libraries with cmake modules
 set(PythonInterp_FIND_VERSION "2")
 find_package(PythonInterp REQUIRED)
-- 
1.9.1

