From 9322fa1768238e3a93520e252c5011d7143ae6e2 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Wed, 18 Jun 2014 16:01:26 +0400
Subject: [PATCH 6/7] Fix python invocation for mavgen (python2)

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7ea8d5e..c1d8128 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,8 @@ set(PROJECT_CONTACT_VENDOR mavlink)
 include(GNUInstallDirs)
 
 # find libraries with cmake modules
-find_package(PythonInterp)
+set(PythonInterp_FIND_VERSION "2")
+find_package(PythonInterp REQUIRED)
 
 # config files
 configure_file(config.h.in config.h)
@@ -30,7 +31,8 @@ macro(generateMavlink version definitions)
         message(STATUS "processing: ${definitionAbsPath}")
         add_custom_command( 
             OUTPUT ${targetName}-stamp
-            COMMAND PYTHONPATH=${CMAKE_SOURCE_DIR} ${PYTHON_EXECUTABLE} -m pymavlink.generator.mavgen --lang=C --wire-protocol=${version}
+            COMMAND /usr/bin/env PYTHONPATH="${CMAKE_SOURCE_DIR}:$$PYTHONPATH"
+                ${PYTHON_EXECUTABLE} -m pymavlink.generator.mavgen --lang=C --wire-protocol=${version}
                 --output=include/v${version} ${definitionAbsPath}
             COMMAND touch ${targetName}-stamp
             DEPENDS ${definitionAbsPath} ${mavgen_path}
-- 
1.9.1

